commands/databases.go

```diff
@@ -1881,30 +1881,60 @@ func databaseConfiguration() *Command {
 			Long:    "The subcommands of `doctl databases configuration` are used to view a database cluster's configuration.",
 		},
 	}
-	getMySQLConfigurationLongDesc := `
+	getConfigurationLongDesc := `
 		This will get a database cluster's configuration given its ID and Engine
 	`
-	getMySQLCfgCommand := CmdBuilder(
+	upadateConfigurationLongDesc := `
+		This will update a database cluster's configuration given its ID and Engine and Desired Configuration (as JSON string)
+	`
```

Comment:
```suggestion
	upadateConfigurationLongDesc := "This will update a database cluster's configuration given its ID, engine, and desired configuration as a JSON string."
```

The way these long descriptions are defined is adding a bit of unneeded whitespace to the help output:

Compare before:
```
$ ./builds/doctl databases configuration update -h

                This will update a database cluster's configuration given its ID and Engine and Desired Configuration (as JSON string)

Usage:
  doctl databases configuration update <db-id> [flags]
```

to after this change: 
```
$ ./builds/doctl databases configuration update -h
This will update a database cluster's configuration given its ID and Engine and Desired Configuration (as JSON string)

Usage:
  doctl databases configuration update <db-id> [flags]
```